package-lock.json

```diff
@@ -3552,61 +3552,100 @@
       }
     },
     "node_modules/@octokit/auth-token": {
-      "version": "2.4.2",
```

Comment:
Why the package-lock changes?

Reply:
It seems the lock was out of sync? maybe some one did not commit previous package lock changes. I was yelled at by github re sync the lock

Reply:
Fair enough.